config/routes.rb

```diff
@@ -34,6 +34,7 @@
   resource :feedback, controller: 'feedback'
 
   namespace :editorial do
+    resources :assets, only: [:index, :new, :create]
```

Comment:
I just need the template to render. is it good practice to make a blank placeholder?